resources/zib/zib-BodyWeight.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-BodyWeight" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-BodyWeight" />
+  <name value="ZibBodyWeight" />
+  <title value="zib BodyWeight" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Body weight is the common name for (human) body mass." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) BodyWeight v3.2(2020)](https://zibs.nl/wiki/BodyWeight-v3.2(2020EN)).&#xD;&#xA;&#xD;&#xA;This profile tries to be compatible with, though is not derived from, the relevant [HL7 Vital Signs](https://www.hl7.org/fhir/observation-vitalsigns.html) profile." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bodyweight-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/BodyWeight-v3.2(2020EN)" />
+    <name value="zib BodyWeight-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BodyWeight" />
+      <alias value="Lichaamsgewicht" />
+      <mapping>
+        <identity value="zib-bodyweight-v3.2-2020EN" />
+        <map value="NL-CM:12.1.1" />
+        <comment value="BodyWeight" />
+      </mapping>
+    </element>
+    <element id="Observation.code.coding">
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="pattern" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.code.coding:bodyWeightCode">
+      <path value="Observation.code.coding" />
+      <sliceName value="bodyWeightCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCoding>
+        <system value="http://loinc.org" />
+        <code value="29463-7" />
+      </patternCoding>
+    </element>
```

Comment:
FHIR specs

"The element Observation.code.coding is sliced based on the values of value:code, value:system"

I don't see a confirmed way that we do the "Vital signs" way in the guidelines: https://informatiestandaarden.nictiz.nl/wiki/FHIR:Vissue-Github-62_FHIR_Profiling_Guidelines_R4#HL7_Vital_Signs